arch/arm64/boot/dts/rockchip/overlays/rk3399-spi1-cs1-slb9670.dts

```diff
@@ -0,0 +1,40 @@
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/pinctrl/rockchip.h>
+
+/ {
+    metadata {
+        title = "Enable Infineon SLB9670 TPM on SPI1";
+        category = "misc";
+        compatible = "rockchip,rk3399";
+        exclusive = "GPIO1_B2", "GPIO4_D5";
+        description = "Enable Infineon SLB9670 Trusted Platform Module (TPM) on SPI1 CE1.\n It configures two GPIO chip selects on SPI1, disconnecting the native one to keep the compatiblity.\nCE0=GPIO1_B2, CE1=GPIO4_D5.";
+    };
+};
+
+&spi1 {
+    status = "okay";
+    #address-cells = <1>;
+    #size-cells = <0>;
+    pinctrl-0 = <&spi1_clk &spi1_tx &spi1_rx &spi1_cs0 &spi1_cs1>;
+    cs-gpios = <&gpio1 RK_PB2 RK_FUNC_GPIO>, <&gpio4 RK_PD5 RK_FUNC_GPIO>;
```

Comment:
```suggestion
    cs-gpios = <&gpio4 RK_PD5 RK_FUNC_GPIO>;
```
Currently `metadata.exclusive` would prevent other SPI1 overlays from enabled. So let's not pretend there is other devices we need to support.

Reply:
As mentioned earlier, some boards, such as the Rock Pi 4c, come with SPI flash soldered on board, and pins 19, 21, 23, and 24 are also routed to it. Although the latter is not exposed on the GPIO header, according to the [official documentation](https://docs.radxa.com/en/rock4/hardware/rock4-gpio), it should still be accessible for SPI flash access.

However, based on my experience, if the native CS functionality is not disabled and replaced with the GPIO-based one, conflicts arise with our attempt to add a second CS. This [article](https://lorenz.brun.one/spi-on-rock64/) confirmed my observations, and thanks to it, I was able to follow the described trick to resolve the issue.

Therefore, I can also confirm that making this modification causes the overlay to stop working.

Reply:
If you combined with pinctrl change it should work. SPI controller will ignore the SPI flash since no CS pin for it was defined.